gpii/node_modules/flowManager/src/Capture.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * GPII snapshot Handler
+ *
+ * Copyright 2014 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/gpii/universal/LICENSE.txt
+ */
+"use strict";
+
+var fluid = require("infusion");
+
+fluid.defaults("gpii.flowManager.capture.handler", {
```

Comment:
There's no need for the capture tool to be exposed over HTTP. This is a remnant of our "endpoints for everything" model of c. 2012 but anything that we expect to be able to invoke this will be part of the same process, i.e. the gpii-app. In general we are trying to reduce localhost endpoints on the client to the absolute minimum anyway as part of https://issues.gpii.net/browse/GPII-3078